tests/models/test_models_unet_1d.py

```diff
@@ -64,7 +64,7 @@ def test_outputs_equivalence(self):
 
     @unittest.skipIf(torch_device == "mps", "mish op not supported in MPS")
     def test_from_pretrained_save_pretrained(self):
```

Comment:
```suggestion
    def test_from_save_pretrained(self):
```

Maybe try this to actually skip the test on `mps`? I haven't tested it yet, if it works we should do the same rename in `UNetRLModelTests`.